server/FuzzServer/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP Sample Server cmake build script
+#
+# Copyright 2012 Marc-Andre Moreau <marcandre.moreau@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(MODULE_NAME "fuzz-server")
```

Comment:
Test name must be with prefix TestFuzz, otherwise oss-fuzz will not run it, see [^1].

[^1]: https://github.com/google/oss-fuzz/blob/cd48a6e2415fc52bf8d8cc155ed61f4b7711e160/projects/freerdp/build.sh#L82